docs/readme-sync/deploy-as-a-microservice/040 - configure-optimizely-agent/050 - agent-plugins.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Agent plugins"
+excerpt: ""
+slug: "agent-plugins"
+hidden: false
+metadata:
+  title: "Agent plugins - Optimizely Full Stack"
+createdAt: "2020-09-21T20:30:00.000Z"
+updatedAt: "2020-09-21T20:30:00.000Z"
+---
+
+## Agent Plugins
+
+Optimizely Agent can be extended through the use of [plugins](https://docs.developers.optimizely.com/full-stack/docs/agent-plugins). Plugins are distinct from the standard Agent packages
```

Comment:
This is a circular/self-referential link; delete.

Reply:
Hey, by the way @mikecdavis  , I set up a preview in the travis stage so you can actually view the published agent-plugin page here before merge to master:

https://rollouts-sandbox-doc-test.readme.io/docs/agent-plugins

(I'm thinking of changing the sandbox domain, FYI)

Reply:
Hey, by the way @mikecdavis  , I set up a preview in the travis stage so you can actually view the published agent-plugin page here before merge to master:

https://rollouts-sandbox-doc-test.readme.io/docs/agent-plugins

(I'm thinking of changing the sandbox domain, FYI...not sure you can see it now without being logged into ReadMe)